libs/contrib/nanovg_helper.xtm

```diff
@@ -0,0 +1,742 @@
+;;; nanovg_helper.xtm -- nanovg helper library
+
+;; Author: Yuya Chiba (cy.blue.9@gmail.com)
+;; Keywords: extempore
+;; Required dylibs: libnanovg
+
+;;; Commentary:
+
+;; This library is warrper library of nanovg(libs/external/nanovg.xtm)
+
+;;; Code:
+
+(sys:load "libs/aot-cache/nanovg_helper.xtm" 'quiet)
+(sys:load-preload-check 'nanovg_helper)
+(define *xtmlib-nanovg_helper-loaded* #f)
+
+(impc:aot:suppress-aot-do
+ (sys:load "libs/external/nanovg.xtm"))
+(impc:aot:insert-forms
+ (sys:load "libs/external/nanovg.xtm" 'quiet))
+
+(impc:aot:insert-header "xtmnanovg_helper")
+
+;; set up the current dylib name and path (for AOT compilation)
```

Comment:
you don't need the `bind-dylib` here - it's already loaded when you load the main nanovg library above.

Reply:
That makes sense.
I'll fix.